### PR TITLE
Fix bug for erase while traversing

### PR DESCRIPTION
**Pull request by xiaocai2333**
issue: #26539

### CODE REVIEW

File: internal/core/src/query/visitors/ExecExprVisitor.cpp [hunk 1]
```diff
@@ -2358,19 +2352,16 @@ ExecExprVisitor::ExecJsonContainsAllArray(JsonContainsExpr& expr_raw)
             for (auto&& e : val) {
                 json_array.emplace_back(e);
             }
-            for (auto index : tmp_elements_index) {
+            for (int index = 0; index < elements.size(); ++index) {
                 if (compareTwoJsonArray(json_array, elements[index])) {
```
Comment (zhagnlu) [inline internal/core/src/query/visitors/ExecExprVisitor.cpp:2356]: name follow same format, maybe CompareTwo.. is better.

**Comment by xiaocai2333** [comment]
rerun ut

File: internal/core/src/query/visitors/ExecExprVisitor.cpp (hunk 1 above)
Comment (xiaocai2333) [reply internal/core/src/query/visitors/ExecExprVisitor.cpp:2356]: it's just an internal function.

**Pull request merged**